extensions/wikia/CreateNewWiki/CreateWikiChecks.php

```diff
@@ -105,6 +105,9 @@ public static function checkDomainIsCorrect( $sName, $sLang, $type = false, $use
 			$message = wfMessage( 'autocreatewiki-violate-policy' );
 		} elseif ( !in_array( 'staff', $app->wg->user->getGroups() ) && (static::checkBadWords( $sName, "domain" ) === false) ) {
 			#-- invalid name (bad words)
+			$message = wfMessage('autocreatewiki-violate-policy');
+		} elseif ( !array_key_exists($sLang, WikiaLanguage::getRequestSupportedLanguages() ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This branch's code block is the same as the block for the branch on line 103. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)